packages/moonstone/Input/InputSpotlightDecorator.js

```diff
@@ -27,6 +27,24 @@ const handleKeyDown = handle(
 	call('onKeyDown')
 );
 
+
+/**
+ * Default config for {@link moonstone/Input.InputSpotlightDecorator}
```

Comment:
Please add one empty line after this line.

Reply:
Fixed